ProjectCoimbra.UWP/Project.Coimbra/Helpers/SongPagesHelper.cs

```diff
@@ -185,5 +184,31 @@ public static async Task FillListBoxAsync(ListBox listBox, string selectedValue,
                 nextButton.IsEnabled = true;
             }
         }
+
+        /// <summary>
+        /// Removes Item from list
+        /// </summary>
+        /// <param name="key">item key</param>
+        public static async Task RemoveFile(string key)
+        {
+            if (key == null)
+            {
+                return;
```

Comment:
In the case of this and other failure cases below, is it OK to just silently fail? Would we need to return additional info to the caller.

Reply:
I'll add some error messages to inform users of the status